docs/pages/getting-data-with-pelican/client.mdx

```diff
@@ -241,6 +241,91 @@ The Pelican client supports a variety of command line flags that modify the clie
 - **-r or --recursive:** Takes no argument and indicates to Pelican that all sub paths at the level of the provided namespace should be copied recursively. This option is only supported if the origin supports the WebDav protocol.
 - **-t or --token:** Takes a path to a file containing a signed JWT, and is used to download protected objects.
 
+## Listing Objects on an Origin
+
+To use the pelican client to list objects in a collection, use Pelican's `object ls` sub-command
+
+```bash
+pelican object ls pelican://<federation-url></namespace-prefix></path/to/collection>
+```
+
+This will only be successful when listings are enabled in the Pelican origin. To learn more about origin capabilities, go to the [Origin docs](../federating-your-data/origin.mdx).
+
+If listings are not enabled, you will see the following error:
+
+```
+ERROR[2025-02-07T15:27:05-06:00] Failure getting pelican://osdf-director.osg-htc.org:443/nrao-ardg: failed to perform list request: Collections URL not found in director response. Are you sure there's an origin for prefix /<namespace-prefix> that supports listings?
+```
+
+## Getting Object Data From a Federation with Stat
+
+To use the pelican client to get object data, use Pelican's `object stat` sub-command.
+
+```bash
+pelican object stat pelican://<federation-url></namespace-prefix></path/to/object>
+```
+
+This will return data in the form of:
+
+```
+Name: </namespace-path></path/to/object>
+Size: <size>
+ModTime: <time-last-modified>
+IsCollection: false
+```
+
+This command can also be run on a collection:
```

Comment:
I grepped through our docs, and I couldn't find anywhere that we actually define what it means to be a collection. Since this documentation explicitly describes how to work with collections, can you also take a stab at defining collections? I think the most appropriate place for the description will be in `docs'/pages/about-pelican/core-concepts.mdx`, right after the current definition of "Objects".

Once there's a definition, can you also link to it from this section?